db/schema.rb

```diff
@@ -319,7 +320,6 @@
     t.string   "unitsLabel",              limit: 255
     t.string   "unitsLabelShort",         limit: 255
     t.integer  "unit_id",                 limit: 4
-    t.integer  "units",                   limit: 4,     default: 1,     null: false
```

Comment:
Certainly not... I thought of it as just one of those weird things that sometimes shows up in the schema.db when doing the migrate... but now I see it's more "insidious". It's changing back something we just changed. Must be because of whatever weird thing happened to cause this problem in first place. Will try to fix.